pkg/infrastructure/clusterapi/clusterapi.go

```diff
@@ -191,6 +200,7 @@ func (i *InfraProvider) Provision(ctx context.Context, dir string, parents asset
 	logrus.Infof("Waiting up to %v (until %v %s) for network infrastructure to become ready...", timeout, untilTime.Format(time.Kitchen), timezone)
 	var cluster *clusterv1.Cluster
 	{
+		// TODO: Fix this to handle both clusters.  The cluster IDs are cleared when doing `create manifests` before `create cluster`
```

Comment:
Comment?

Reply:
I'll remove.  thanks!

Reply:
@jcpowermac , I updated this logic to now to loop and check each infra cluster.